.vscode/launch.json

```diff
@@ -278,6 +278,7 @@
       "request": "launch",
       "name": "Tests: mds-db",
       "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+      "runtimeArgs": ["--harmony"],
```

Comment:
I think you added this to try and get `async/await` working nicely in the debugger, but IIRC it didn't work? Might as well remove

Reply:
will do